### PR TITLE
Add missing SpaceHeatingLoad and DomesticHotWaterLoad inputs and ExistingBoiler outputs

### PR DESCRIPTION
**Pull request by adfarth**
### Please check if the PR fulfills these requirements
- [ ] CHANGELOG.md is updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?
(Bug fix, feature, docs update, ...)

Added: 
- Previously missing `SpaceHeatingLoad` and `DomesticHotWaterLoad` inputs: `addressable_load_fraction` 
- Previously missing `ExistingBoiler` outputs: `year_one_thermal_to_tes_series_mmbtu_per_hour` and `year_one_fuel_consumption_series_mmbtu_per_hour` 

Removed: 
- `ExistingBoiler` output: `thermal_to_tes_series_mmbtu_per_hour`, `year_one_fuel_consumption_mmbtu_per_hour`, `year_one_thermal_production_mmbtu_per_hour` 

### What is the current behavior?
(You can also link to an open issue here)

### What is the new behavior (if this is a feature change)?
- Add `addressable_load_fraction` inputs (and validation) for `SpaceHeatingLoad` and `DomesticHotWaterLoad` which effectively ignores a portion of the entered loads. These inputs can be scalars (applied to all time steps of the year), monthly (applied to the timesteps of each month), or of length 8760 * `time_steps_per_hour`. 
     - We handle the alignment of `addressable_load_fraction` and the fuel input (`annual_mmbtu`, `monthly_mmbtu`, or `fuel_loads_mmbtu_per_hour`) in REopt.jl Julia package. 

### Does this PR introduce a breaking change?
(What changes might users need to make in their application due to this PR?)

### CODE REVIEW

File: job/models.py [hunk 1]
```diff
@@ -4044,7 +4054,7 @@ def clean(self):
         
         pass
 
-# TODO Add domestic hot water input model.
+# TODO Add domestic hot water input model. - done?
```
Comment (adfarth) [inline job/models.py:4057]: @rathod-b  This is complete now, right?

Comment (adfarth) [reply job/models.py]: Bhavesh relayed that it is and I deleted this line.

File: job/models.py [hunk 2]
```diff
@@ -3579,6 +3575,16 @@ class ExistingBoilerOutputs(BaseModel, models.Model):
         default = list,
     )
 
+    year_one_thermal_to_tes_series_mmbtu_per_hour = ArrayField(
```
Comment (adfarth) [inline job/models.py:3578]: @rathod-b do you know why there are two `thermal_to_tes_series_mmbtu_per_hour` outputs for ExistingBoiler?

**Comment by adfarth** [comment]
> @adfarth Is this ready for review? I just want to push this through since it's been a while. I can review if need be. Looks like we should just merge develop into here, see if there are any merge conflicts, and resolve if needed. I can help with that.

@Bill-Becker sorry--this totally slipped through the cracks! Yes, I believe it's ready for review. One main thing to check/issue we might encounter is that, as far as I know, we're unable to provide two model field types in with Django the same way we can specify `addressable_load_fraction::Union{<:Real, AbstractVector{<:Real}}` in Julia. So, I think one could not set `addressable_load_fraction = 1.0` but would have to use `=[1.0]` in the API.

**Comment by Bill-Becker** [comment]
@adfarth We can close this PR without merging if you are good with the new PR https://github.com/NREL/REopt_API/pull/398 that I made to replicate the changes here.

**Comment by adfarth** [comment]
> @adfarth We can close this PR without merging if you are good with the new PR #398 that I made to replicate the changes here.

@Bill-Becker yes sounds good to me!

**Pull request closed without merging**